PROJECTS.md

```diff
@@ -0,0 +1,48 @@
+# OpenJS Foundation Projects
+
+The projects below are full OpenJS Foundation members at the stage level indicated in the heading. Projects may move to new stages upon annual review or by request. 
+
+For more information on the stage definitions and how projects move between stages, please see the [Project Progression Document](/PROJECT_PROGRESSION.md). Links will navigate you to the project's official website.
+
+## Impact Projects
+
+* [Appium](http://appium.io/)
+* [Dojo](https://dojo.io/)
+* [jQuery](https://jquery.com/)
+* [Node.js](https://nodejs.dev/)
```

Comment:
The link should be https://nodejs.org/, no?

Reply:
```suggestion
* [Node.js](https://nodejs.org/)
```

Reply:
Whatever ya'll prefer! I wondered if you'd like .dev site because its new and shiny? No matter to me!

Reply:
The .dev site is a tutorial. The .org site is where you go to download Node.js, read the reference documentation, etc. I would definitely prefer the .org site.